src/components/DisplayNames/DisplayNamesWithTooltip.js

```diff
@@ -51,7 +51,7 @@ function DisplayNamesWithToolTip(props) {
     return (
         // Tokenization of string only support prop numberOfLines on Web
         <Text
-            style={[...props.textStyles, styles.pRelative]}
+            style={[...props.textStyles, styles.pRelative, props.numberOfLines === 1 ? styles.noWrap : {}]}
```

Comment:
`styles.noWrap` is not needed here https://github.com/Expensify/App/issues/32555#issuecomment-1842636943